components/Buttons/src/Theming/MDCFloatingButton+MaterialTheming.m

```diff
@@ -55,7 +54,18 @@ - (void)applySecondaryThemeWithShapeScheme:(id<MDCShapeScheming>)scheme {
 }
 
 - (void)applySecondaryThemeWithTypographyScheme:(id<MDCTypographyScheming>)scheme {
-  [MDCButtonTypographyThemer applyTypographyScheme:scheme toButton:self];
+  [self resetTitleForAllStates];
+  [self setTitleFont:scheme.button forState:UIControlStateNormal];
+}
+
+#pragma mark - General helpers
+
+- (void)resetTitleForAllStates {
```

Comment:
Maybe this should be named `resetTitleFontForAllStates`

Reply:
Yeah that's a much better name, I updated it in the PR.